lib/srv/db/proxyserver.go

```diff
@@ -211,7 +211,9 @@ func (s *ProxyServer) ServePostgres(listener net.Listener) error {
 			defer clientConn.Close()
 			err := s.PostgresProxy().HandleConnection(s.closeCtx, clientConn)
 			if err != nil {
-				s.log.WithError(err).Warn("Failed to handle Postgres client connection.")
+				if !utils.IsOKNetworkError(err) {
+					s.log.WithError(err).Warn("Failed to handle Postgres client connection.")
+				}
 			}
```

Comment:
Maybe:

```suggestion:
			if err != nil && !utils.IsOKNetworkError(err)  {
				s.log.WithError(err).Warn("Failed to handle Postgres client connection.")
			}
```

Reply:
A comment might be good too, something like: `// Do not log OKNetworkErrors, as it spams the logs`.